spec/models/comment_spec.rb

```diff
@@ -1,5 +1,22 @@
 require 'rails_helper'
 
+# write test for Like model here
 RSpec.describe Comment, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  it 'is invalid without a body' do
+    expect(Comment.new(text: nil)).to be_invalid
+  end
+
+  # test user association with new comment
+  it 'belongs to a user' do
+    user = User.create(name: 'ALen', bio: 'bio')
+    comment = Comment.create(text: 'text', author: user)
+    expect(comment.author).to eq(user)
+  end
+
+  # test post association with new comment
+  it 'belongs to a post' do
+    post = Post.create(title: 'title', author: User.create(name: 'ALen', bio: 'bio'))
```

Comment:
- [ ] Please fix the error.

![image](https://user-images.githubusercontent.com/38635158/161381068-3856b1b0-0a00-4a68-a9b9-b038ff63dc8c.png)